### PR TITLE
New extension: Bitfinex margin funding

### PR DESCRIPTION
**Pull request by Yukaii**
## Description

A Raycast extension that utilizes the Bitfinex [margin funding API](https://docs.bitfinex.com/reference/margin-funding). You can view past interests, create/update/cancel offers, and look up lending rates inside Raycast.

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

**Comment by Yukaii** [comment]
Hi @danielsequeira , I've fixed it in [`55ba1b8`](https://github.com/raycast/extensions/pull/2267/commits/55ba1b83f04645381e84812009f5d7c1472e48aa).

File: extensions/bitfinex/README.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+1. Go to [Bitfinex API keys](https://setting.bitfinex.com/api#my-keys) and create a new API key with the following permission
+   - **Account History**: `Get historical balances entries and trade information`
+   - **Orders**: `Create and cancel orders`
+   - **Margin Funding** permissions
+   - **Wallets**: `Get wallet balances and addresses
+    ![API Permission](media/api_permission.png)
+2. Open the extension command and paste your API key and secret into the fields.
+3. Choose the currency you want to use.
+   ![Extension Preference](media/preference.png)
```
Comment (danielsequeira) [inline extensions/bitfinex/README.md:13]: I would remove the images from the README, they are not properly aligned with the content and I think a good text description in this case is already good enough.

File: extensions/bitfinex/README.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Bitfinex Extension
+
+## Configuration
+
+1. Go to [Bitfinex API keys](https://setting.bitfinex.com/api#my-keys) and create a new API key with the following permission
+   - **Account History**: `Get historical balances entries and trade information`
+   - **Orders**: `Create and cancel orders`
+   - **Margin Funding** permissions
+   - **Wallets**: `Get wallet balances and addresses
```
Comment (danielsequeira) [inline extensions/bitfinex/README.md:9]:
```suggestion
   - **Wallets**: `Get wallet balances and addresses`
```

File: extensions/bitfinex/package.json [hunk 3]
```diff
@@ -0,0 +1,1331 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "bitfinex",
+  "title": "Bitfinex",
+  "description": "A raycast extension built for Bitfinex margin funding",
```
Comment (danielsequeira) [inline extensions/bitfinex/package.json:5]:
```suggestion
  "description": "A Raycast extension built for Bitfinex margin funding",
```

File: extensions/bitfinex/README.md (hunk 1 above)
Comment (Yukaii) [reply extensions/bitfinex/README.md]: okay! 👍

**Review by danielsequeira — APPROVED** [review]

**Pull request merged**